contracts/script/DeployOpenEigenLayer.s.sol

```diff
@@ -7,7 +7,6 @@ import "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.so
 import "@openzeppelin/contracts/proxy/beacon/UpgradeableBeacon.sol";
```

Comment:
Not sure what these changes are and if they're relevant

Reply:
fixes mock deployment of core for unrelated changed